pkg/controller/clustercontroller/backups.go

```diff
@@ -170,3 +188,23 @@ func (j job) Run() {
 			backupName, j.Name, err)
 	}
 }
+
+func backupGC(j job, backups *api.MysqlBackupList) {
```

Comment:
You can change the signature of this function to `func (j *job) backupGC() {}` and do the backup listing here.